src/test/scala/uk/gov/homeoffice/cirium/CiriumStreamToPortResponseSpec.scala

```diff
@@ -56,7 +55,42 @@ class CiriumStreamToPortResponseSpec extends TestKit(ActorSystem("testActorSyste
     }
   }
 
-  class MockClientWithFailures(startUri: String)(implicit system: ActorSystem) extends Cirium.Client("", "", startUri) {
+  class MockClientWithoutRequestObjectInResponse(startUri: String, metricsService: MetricsService)(implicit system: ActorSystem) extends Cirium.Client("", "", startUri, metricsService) {
+
+    val latestRegex: Regex = "https://latest.+".r
+    val previousRegex: Regex = "https://current/previous/.+".r
+    val forward1Regex: Regex = "https://item/1/.+".r
+    val forward3Regex: Regex = "https://item/3/.+".r
+    val forward5Regex: Regex = "https://item/5/.+".r
```

Comment:
Shall we order these 1-5, and make the regex for #4 match the others? (missing a forward slash after '4')